source/_integrations/daikin.markdown

```diff
@@ -95,16 +95,20 @@ _When "Holiday mode" is enabled, the following action take place:_
 
 ## Sensor
 
-The `daikin` sensor platform integrates Daikin air conditioning systems into Home Assistant, enabling displaying the following parameters:
+The `daikin` sensor platform integrates Daikin air conditioning systems into Home Assistant, enabling displaying the following parameters by device:
 
 - Inside temperature
-- Outside temperature
 - Inside humidity
-- Estimated power consumption
 - Hourly energy consumption in cool mode
 - Hourly energy consumption in heat mode
+- Today's total energy consumption (updated hourly, resets at 00:00)
+
+The integration displays the following parameters for the outdoor compressor:
```

Comment:
The outdoor measurements was removed from the PR, right?

Reply:
The "outdoor" device has been removed but not the sensors, and IMO it's still worth distinguishing indoor and outdoor sensors in the doc

Reply:
Ok.